src/xsl/openrosa2html5form.xsl

```diff
@@ -522,10 +530,10 @@ XSLT Stylesheet that transforms OpenRosa style (X)Forms into valid HTMl5 forms
                             <xsl:apply-templates select="xf:label" />
                         </xsl:if>
 
-                        <xsl:if test="not(local-name() = 'item' or local-name() = 'bind' or local-name = 'setvalue')">
-                            <!-- the only use case at the moment is a <setvalue> child with odk-xforms-changed event-->
-                            <xsl:if test="./xf:setvalue[@event]">
-                                <xsl:apply-templates select="./xf:setvalue[@event]" />
+                        <xsl:if test="not(local-name() = 'item' or local-name() = 'bind' or local-name() = 'setvalue' or local-name() = 'setgeopoint')">
+                            <!-- the only use case at the moment is a <setvalue> child with xforms-value-changed event-->
```

Comment:
I think this should now be `a nested action triggered by xforms-value-changed event` or just adding setgeopoint so the comment matches.

Reply:
Ah missed that one! Updated.